src/Compilers/VisualBasic/Test/Semantic/Semantics/LambdaTests.vb

```diff
@@ -2164,5 +2164,223 @@ In getter
 ]]>)
         End Sub
 
+        <WorkItem(53593, "https://github.com/dotnet/roslyn/issues/53593")>
+        <Fact()>
+        Public Sub Issue53593_1()
+
+            Dim compilationDef =
+<compilation>
+    <file name="a.vb">
+Public Class C0
+
+    Public a, b As New C1((Function(n) n + 1)(1))
```

Comment:
>Public a, b As New C1 [](http://example.com/codeflow?start=4&length=21)

Are we testing property initializers as well?

Reply:
> Are we testing property initializers as well?

Issue53593_6, WithEvents fields are actually properties